packages/react/src/components/Card/Card.module.css

```diff
@@ -0,0 +1,154 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  /*
+   * TODO: discuss how we would like to determine max width
+   */
+  width: 330px;
+}
+
+a.card {
+  color: inherit;
+  text-decoration: none;
+}
+
+.card img {
```

Comment:
This would not work if the user sends their own picture down, see comment in `Card.tsx` you might need to account for `picture` as well.
And it should also only select a direct child of `.card` by using `.card > img`